lib/connectors/mongodb/connector.rb

```diff
@@ -110,7 +110,9 @@ def yield_documents
 
             view = cursor.skip(skip).limit(PAGE_SIZE)
             view.each do |document|
-              yield serialize(document)
+              yield_with_handling_transient_errors do
+                yield serialize(document)
```

Comment:
Opinion: It might be good to have a unit/integration for the actual connector.

Reply:
Good call, will add!